jobserver/views/workspaces.py

```diff
@@ -318,6 +294,44 @@ def get(self, request, *args, **kwargs):
             context=context,
         )
 
+    def get_output_permissions(self, user, workspace):
+        is_privileged_user = has_permission(
+            user, "release_file_view", project=workspace.project
+        )
+
+        # a user can see backend files if they have access to at least one
+        # backend and the permissions required to see outputs
+        has_backends = (
+            user.is_authenticated and user.backends.exclude(level_4_url="").exists()
+        )
+        can_view_files = is_privileged_user and has_backends
+
+        # are there any releases to show for the workspace?
+        can_view_releases = workspace.releases.exists()
```

Comment:
When testing this locally, I was able to click on the "Released Outputs" link and go to the releases page, while I was logged out and also as an interactive user.

Clicking on one of the releases produces a 404, but still seems like the link shouldn't be enabled.

Reply:
I can confirm it's fixed, but the behaviour is inconsistent with published outputs, which doesn't have a link if there are no outputs for unprivileged users (it has a link for privileged users though!)

Reply:
For reference, I'm looking at http://localhost:8000/datalab/bmi-short-data-report/bmi-short-data-report/ while logged in and logged out

Reply:
Fixed and a small tweak handled on the published page too